### DISCUSSION TITLE
Outputting the version

### DISCUSSION THREAD

**Discussion by nhedger** [orhun/git-cliff-action#11] [Ideas]
How would you feel about adding a `version` output that would hold the version of the latest release (based on the `context`)?

For example, another step could use it to update the version field in a `package.json` file.

Happy to contribute the feature myself.

--

Assuming you'd be down, here is a question:

- Does the JSON context guarantee the order of the releases (first item is the latest release)?

**Comment by nhedger** [comment]
Sorry, meant to create this in https://github.com/orhun/git-cliff-action

**Reply by orhun** [reply]
Moved!

**Comment by orhun** [comment]
> How would you feel about adding a version output that would hold the version of the latest release

That'd be great!

> Does the JSON context guarantee the order of the releases (first item is the latest release)?

Yup, they are always in order.

**Reply by nhedger** [reply]
Awesome, I'll send a PR!

**Comment by nhedger** [comment]
Sent a PR in #12
